src/components/Label/Label.st.css

```diff
@@ -0,0 +1,37 @@
+:import {
+  -st-from: "wix-ui-core/index.st.css";
+  -st-named: Label, LabelStyle;
+}
+  
+:import {
+  -st-from: "../../palette.st.css";
+  -st-named: secondaryLabel;
+}
+
+:import {
+  -st-from: "../../typography.st.css";
+  -st-named: fontLight;
+}
+
+.root {
+  -st-extends: Label;
+  -st-states: appearance(string);
+}
+
+.root:appearance(T1.1) {
+  -st-mixin: LabelStyle(
```

Comment:
Mixin has the biggest potential as a feature, but it is actually causing us a lot of issues as it is not ready.
Let's drop the mixin here, or at least use it once, and then after override the stuff you need (the color is always  secondaryLabel for example)